CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+- Fix: Move Translator out of template
```

Comment:
I find this too unspecific. Which template? Also: Why? What does it fix? (Of course we don't want to go into too much detail here, but a little context would be helpful)
I think it's a good idea to include the type (fix, chore, etc.) here, but we should agree on it and then use it consistently.

Reply:
I think there is no need in mentioning any implementation detail. If this fixes a bug, it can be mentioned here.

Reply:
Yeah, something like "fix `undefined Translator` error in `componentName`" would do

Reply:
True. changed in 97506fe0ffcecee2fe83ea1af4a4ebed56667983